src/symmeplot/matplotlib/scene.py

```diff
@@ -144,6 +145,51 @@ def plot(self, prettify=True, ax_scale=1.5):
             self.auto_zoom()
             self.axes.set_aspect("equal", adjustable="box")
 
+    def set_plot_as_2d(self, frame: ReferenceFrame | None = None) -> None:
+        """Change the axis to an orhogonal projection making the view seemingly 2D.
+
+        Parameters
+        ----------
+        frame : ReferenceFrame, optional
+            Reference frame w.r.t. which the axis view is oriented aligning the users
+            view with the XY plane. The default is the inertial frame of the scene.
+
+        """
+        projection_frame = ReferenceFrame("A")
+        projection_frame.orient_axis(self.inertial_frame, self.inertial_frame.z, 0)
+        frame = frame or projection_frame
+        self.axes.set_proj_type("ortho")
+        self.axes.view_init(**self.get_euler_angels(self.inertial_frame, frame))
+
+    @staticmethod
+    def get_euler_angels(normal_frame: ReferenceFrame, projection_frame: ReferenceFrame,
```

Comment:
It might be better to change this to a protected method, such that it isn't exposed to the users, i.e. rename to `_get_euler_angles`.

Reply:
Sounds good.